tests/Infrastructure/Vine/GetActiveFiltersTest.php

```
@@ -0,0 +1,197 @@
+
+            $this->assertEquals(1, $activeFilters->total());
+            $this->assertEquals('taxon-2', $activeFilters->first()->getId());
+        }
+    }
+
+    public function test_it_keeps_main_taxon_as_filter_when_the_active_filters_are_no_children_of_main_taxon()
+    {
+        foreach (Catalog::drivers() as $catalog) {
+
+            // Create catalog
+            $taxonomy = $catalog->createTaxonomy();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$taxonomy'. (undefined)

(UnusedLocalVariable)